es63x/src/main/scala/tech/beshu/ror/es/utils/XPackSecurityAuthenticationHeader.scala

```diff
@@ -53,7 +53,7 @@ object XPackSecurityAuthenticationHeader {
       output.writeString(userName)
     } else {
       output.writeString(userName)
-      output.writeStringArray(Array("kibana_admin"))
```

Comment:
I think we need only a superuser role because this role contains all kibana_admin permissions (it's something like unrestricted kibana_access)

Reply:
ok, I've tested in our sandbox and using ECK PoC and it seems you are right